app/src/main/AndroidManifest.xml

```diff
@@ -9,11 +9,17 @@
         android:icon="@mipmap/ic_launcher"
         android:label="@string/app_name"
         android:roundIcon="@mipmap/ic_launcher_round"
-        android:supportsRtl="true"
+        android:supportsRtl="false"
```

Comment:
Our App language is English?

Reply:
i dont think its matter
its only change the position of objects